### ISSUE TITLE
Add Lilypond export functionality

### ISSUE THREAD

**Issue by dpsanders** [dpsanders/MusicTheory.jl#14]

**Comment by NeroBlackstone** [comment]
I think first we need a container like [voice](https://www.w3.org/2021/06/musicxml40/musicxml-reference/elements/voice/) to group note sequences

**Comment by dpsanders** [comment]
Agreed. This is in progress [here](https://github.com/dpsanders/MusicMakie.jl/blob/main/src/parts.jl). Despite the name of that package, it is really more like `MusicTypesetting.jl` with a Makie backend.

Unfortunately I don't have much spare time to work on these packages but any contributions are very welcome!
